src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -24,23 +24,20 @@ static inline int MPIDI_OFI_win_progress_fence_impl(MPIR_Win * win, bool do_free
 {
     int mpi_errno = MPI_SUCCESS;
     int itercount = 0;
-    int ret;
-    uint64_t tcount, donecount;
+    int ret, max_cq_reads;
```

Comment:
`max_ofi_progress` is probably a better naming? One call to `NM_progress` may contain multiple cq reads.

Reply:
Makes sense, `max_ofi_progress` is better.